backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/badge/BpkBadge.kt

```diff
@@ -102,11 +117,11 @@ private val BpkBadgeType.contentColor: Color
 private val BpkBadgeType.backgroundColor: Color
     @Composable
     get() = when (this) {
-        BpkBadgeType.Normal -> BpkTheme.colors.surfaceHighlight
```

Comment:
Looking at the new screenshots seems that we also need to update the `contentColor` to `textOnDark` for the Success, Warning and Destructive. See: 
![image](https://github.com/Skyscanner/backpack-android/assets/4923867/e3a6052c-055f-467c-9016-7e838762eaf1)

Reply:
Thanks, I've fixed this now!